Application/EdFi.Admin.DataAccess/Models/Application.cs

```diff
@@ -42,12 +40,12 @@ public Application()
 
         public virtual ICollection<Profile> Profiles { get; set; }
 
-        public ApplicationEducationOrganization CreateApplicationEducationOrganization(int educationOrganizationId)
+        public ApplicationEducationOrganization CreateApplicationEducationOrganization(long educationOrganizationId)
```

Comment:
This changed to `long` in ODS/API 7.2, so I see why you copied it over. But we might not want to do this.

@vimayya do you think we should change to `long` in ODS/API 6.2?